main.py

```diff
@@ -1,7 +1,38 @@
 import init_django_orm  # noqa: F401
 
 from django.db.models import QuerySet
+from db.models import Genre, Actor
 
 
 def main() -> QuerySet:
-    pass
+    genres = ("Western", "Action", "Dramma")
+    actors = (
+        ("George", "Klooney"),
+        ("Kianu", "Reaves"),
+        ("Scarlett", "Keegan"),
+        ("Will", "Smith"),
+        ("Jaden", "Smith"),
+        ("Scarlett", "Johansson")
+    )
+    for genre in genres:
+        Genre.objects.create(
+            name=genre
+        )
+    for actor in actors:
+        Actor.objects.create(
+            first_name=actor[0],
+            last_name=actor[1]
+        )
```

Comment:
Here you should use unboxing, it will make your code more readable
```suggestion
    for first_name, last_name  in actors:
        Actor.objects.create(
            first_name=first_name,
            last_name=last_name
        )
```

Reply:
Thanks!